includes/class-base-widget.php

```diff
@@ -397,9 +397,98 @@ protected function render_form_textarea( array $field ) {
 	}
 
 	/**
+	 * Render the 'Day' div and select fields
+	 *
+	 * @param  array $field Field data
+	 * @param  array $day   The current day in the iteration.
+	 * @param  array $hours The array of times.
+	 *
+	 * @since NEXT
+	 *
+	 * @return mixed
+	 */
+	protected function render_day_input( $field, $day, array $hours ) {
+
+		$field['name'] = str_replace( 'value', strtolower( $day ), $field['name'] );
+
+		$field['disabled'] = ( $hours['not_open'] ) ? true : false;
+
+		printf(
+			'<div class="day-container">%1$s<p>%2$s</p></div>',
+			'<strong>' . esc_html( ucwords( $day ) ) . '<span class="day-checkbox-toggle"><input name="' . $field['name'] . '[not_open]" id="' . $field['name'] . '[not_open]" class="js_wphoow_closed_checkbox" type="checkbox" value="1" ' . $this->checked( $hours['not_open'], true ) . '><label for="' . $field['name'] . '[not_open]" class="js_wphoow_closed_checkbox"><small>' . esc_html__( 'Closed', 'hours-of-operation-widgets' ) . '</small></label></strong></span>',
+			$this->render_hours_selection( $field, sanitize_title( $day ), $hours )
+		);
+
+	}
+
+	/**
+	 * Render the 'Hours' select fields
+	 *
+	 * @param array $field Field data
+	 * @param array $day   The current day in the iteration.
+	 * @param array $hours The array of times.
+	 *
+	 * @since NEXT
+	 *
+	 * @return mixed
+	 */
+	protected function render_hours_selection( $field, $day, $hours ) {
+
+		ob_start();
+
+		$times = $this->get_half_hour_time_array();
+
+		$field['name'] = str_replace( 'value', strtolower( $day ), $field['name'] );
+
+		$disabled_field = $field['disabled'] ? ' disabled="disabled"' : '';
+
+		echo '<select name="' . esc_attr( $field['name'] . '[open]' ) . '"' . $disabled_field . '>';
```

Comment:
@EvanHerman Seems like there is a lot of `echo` in this block. It might look nicer if we close the php tag and use html directly.